lib/services/presets.go

```diff
@@ -158,3 +159,49 @@ func NewPresetAuditorRole() types.Role {
 	role.SetLogins(types.Allow, []string{"no-login-" + uuid.New().String()})
 	return role
 }
+
+// DefaultAllowRules has the Allow rules that should be set as default when they were no explicitly defined.
+// This is used to update the current cluster roles when deploying a new resource.
+func DefaultAllowRules() map[string][]types.Rule {
+	return map[string][]types.Rule{
+		teleport.PresetEditorRoleName: {
+			types.NewRule(types.KindConnectionDiagnostic, RW()),
```

Comment:
can you add a comment at the place someone would define a new rule that they should probably update this?

Reply:
Yeah, great idea
I'll add a comment on each of the presets
I'll probably add a comment at the bottom of the list of rules (that's where developers usually add the new rules)